.github/workflows/tests-pro-integration.yml

```diff
@@ -65,7 +65,7 @@ on:
       - 'v[0-9]+'
       - release/*
   schedule:
-    - cron: '0 4 * * *'  # run once a day at 4 AM UTC
+    - cron: '15 4 * * *'  # run once a day at 4:15 AM UTC
```

Comment:
I figured it would be good to shift the schedule away from the round hour as we did with ext, so that we don't run into issues with resource availability

Reply:
Well, okay, seems unrelated, and I haven't seen any of these issues yet, but we can just leave this here 😄  😛